modules/ballerina-builtin/src/main/ballerina/ballerina/util/natives.bal

```diff
@@ -46,3 +46,9 @@ public native function getHmacFromBase64 (string baseString, string keyString, s
 @Param { value:"algorithm: The hashing algorithm to be used" }
 @Return { value:"The hashed string" }
 public native function getHmac (string baseString, string keyString, string algorithm) (string);
+
+@doc:Description { value:"Returns validity of the jwt "}
+@doc:Param { value:"jwToken: JW Token" }
+@doc:Param { value:"publicKey: public key " }
+@doc:Return { value:"boolean: validity" }
+native function getShaWithRsa (string jwToken, string publicKey) (boolean);
```

Comment:
This does not return a hash, so the `getSha` part in the name can be misleading. I think it would be better to change this to `verifyJwt` or something similar to that. WDYT?

Reply:
Yes, that's also a good idea. But my suggestion is if we change this to `decryptRsa()` kind of thing it would be more useful than restrict it only to JWT verification. Referring this, can we add full RSA encryption and decrypt support to Ballerina? because it would become more helpful for others occasions as well. 

If we need to add a function like `verifyJwt()` then we have to support other JWT verification algorithms as well.